src/mscorlib/CreateRuntimeRootILLinkDescriptorFile.targets

```diff
@@ -0,0 +1,28 @@
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <CompileDependsOn>_CreateILLinkRuntimeRootDescriptorFile;$(CompileDependsOn)</CompileDependsOn>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <_ILLinkRuntimeRootDescriptorFileName Condition=" '$(_ILLinkRuntimeRootDescriptorFileName)' == '' ">System.Private.CoreLib.xml</_ILLinkRuntimeRootDescriptorFileName>
+    <_ILLinkRuntimeRootDescriptorFilePath Condition=" '$(_ILLinkRuntimeRootDescriptorFilePath)' == '' ">$(IntermediateOutputPath)$(_ILLinkRuntimeRootDescriptorFileName)</_ILLinkRuntimeRootDescriptorFilePath>
+    <_NamespaceFilePath Condition=" '$(_NamespaceFilePath)' == '' ">$(MSBuildThisFileDirectory)..\vm\namespace.h</_NamespaceFilePath>
+    <_MscorlibFilePath Condition=" '$(_MscorlibFilePath)' == '' ">$(MSBuildThisFileDirectory)..\vm\mscorlib.h</_MscorlibFilePath>
+    <_CortypeFilePath Condition=" '$(_CortypeFilePath)' == '' ">$(MSBuildThisFileDirectory)..\inc\cortypeinfo.h</_CortypeFilePath>
+    <_RexcepFilePath Condition=" '$(_RexcepFilePath)' == '' ">$(MSBuildThisFileDirectory)..\vm\rexcep.h</_RexcepFilePath>
+  </PropertyGroup>
+
+  <!-- Generates the xml root descriptor file for ILLink. The file contains roots required by the runtime. -->
+  <UsingTask TaskName="CreateRuntimeRootILLinkDescriptorFile" AssemblyFile="$(MSBuildThisFileDirectory)..\..\packages\illink.tasks\$(ILLinkTasksPackageVersion)\tools\net46\ILLink.Tasks.dll" />
+  <Target Name="_CreateILLinkRuntimeRootDescriptorFile"
+          Inputs="$(_NamespaceFilePath);$(_MscorlibFilePath);"
+          Outputs="$(_ILLinkRuntimeRootDescriptorFilePath);$(_CortypeFilePath);$(_RexcepFilePath)">
```

Comment:
Should _CortypeFilePath and _RexcepFilePath be inputs?

Reply:
Yes, they should be. Updated.